### PR TITLE
Update setup.mdx

### PR DESCRIPTION
**Pull request by wtlin1228**
Add customized `screen` and `within` for using custom queries globally.

### CODE REVIEW

File: docs/react-testing-library/setup.mdx [hunk 1]
```diff
@@ -247,44 +247,76 @@ export {
 You can then override and append the new queries via the render function by
 passing a [`queries`](api.mdx#render-options) option.
 
-If you want to add custom queries globally, you can do this by defining a custom
-render method:
+If you want to add custom queries globally, you can do this by defining your customized
+`render`, `screen` and `within`:
```
Comment (timdeschryver) [inline docs/react-testing-library/setup.mdx:251]:
```suggestion
If you want to add custom queries globally, you can do this by defining your customized
`render`, `screen` and `within` methods:
```

**Review by timdeschryver — APPROVED** [review]

File: docs/react-testing-library/setup.mdx [hunk 2]
```diff
@@ -247,44 +247,76 @@ export {
...
 
 ```js title="test-utils.js"
-import {render, queries} from '@testing-library/react'
+import {render, queries, screen, within} from '@testing-library/react'
 import * as customQueries from './custom-queries'
 
+const customScreen = {
+  ...screen,
+  ...within(document.body, customQueries),
+}
```
Comment (eps1lon) [inline docs/react-testing-library/setup.mdx:265]: I think this should move below `allQueries` and then 
```suggestion
const customScreen = within(document.body, allQueries)
```

Comment (wtlin1228) [reply docs/react-testing-library/setup.mdx]: Done

**Review by eps1lon — APPROVED** [review]
Thanks!

**Comment by eps1lon** [comment]
@all-contributors add @wtlin1228 for docs

**Pull request merged**